cogs/alias.py

```diff
@@ -158,8 +162,9 @@ def remove_old(self):
     def first_word(self, msg):
         return msg.split(" ")[0]
 
-    def get_prefix(self, msg):
-        for p in self.bot.command_prefix:
+    def get_prefix(self, server, msg):
```

Comment:
If you take a look at the calls to `get_prefix` in alias you'll see that most of the time there's no  message object to be passed. The call in `on_message` seem to be the only exception. I'll leave it as it is.